docs/general-reference/help-button.md

```diff
@@ -0,0 +1,113 @@
+    1. **Firebolt components**- the status of the following Firebolt elements:
+        * *Firebolt's Web Application* - availability of Firebolt's web application 
+        * *Firebolt's API*  - availability of Firebolt's API interface
+        * *Firebolt's Engines* - availability of the elements responsible for Engines create, read, and delete related actions
+        * *Firebolt Databases* - availability of the elements responsible for Database creation, read, update and delete related actions
+        * *Firebolt Tenant Account Creation* - availability of the elements responsible for provisioning Firebolt's accounts in AWS
+    2. **AWS components** - the status of the following AWS components, as related to Firebolt availability:
+        * *EC2* - availability of [AWS EC2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html)  
+        * *S3* - availability of [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html)
+        * *Auto Scaling Group (ASG)* - availability of [Auto Scaling Group](https://docs.aws.amazon.com/autoscaling/ec2/userguide/auto-scaling-groups.html)
+        * *AWS regions (ec2-us-east1, ec2-us-east-2, ec2-eu-west-1, etc.)* - availability of different [AWS regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)
+
+2. **Past Incidents** - present critical incidents and their status
+
+#### Historical Uptime
+
+![Status Page](../assets/images/status_page_click.png)
+
+The historical uptime page presents historical information on availability and incidents. Click on the **View Historical Uptime** link in the **Status page** to access the page. 
+
+The page contains two tabs:
+1. **Uptime** - allows you to view availability across different timelines. You can click on a particular day to dive deep into components availability at that date. 
+
+![Historical Status](../assets/images/historical_status.png)
+
+2. **Incidents** - allows you to view past incidents across different timelines. You can click on a particular incident to dive deep into the status of that incident.
+
+![Historical Incidents](../assets/images/historical_incidents.png)
+
+
+#### Subscribe
+You can subscribe to notifications whenever Firebolt **creates**, **update** or **resolve** incidents. You can subscribe to notifications via the following options:
+1. **Email notification** - by providing your email address
+2. **Text notifications** - by providing your phone number. 
+*Please note that text messages will only notify you when Firebolt creates or resolves an incident.*
+3. **Slack notifications** - by connecting via Slack.
+*Please note - you can also get maintenance status through Slack* 
+4. **RSS updates** - by connecting via your RSS/Atom application
+
+### Contact Support
+Contact Support allows you to create a support case for Firebolt's support team.
+
+![Create Case](../assets/images/create_support_case.png)
```

Comment:
I would not use your real email in this image .... @example.net works great as a demo

Reply:
Done